pysatModels/models/sami2py_sami2.py

```diff
@@ -133,12 +134,21 @@ def load(fnames, tag=None, inst_id=None, **kwargs):
     """
 
     # Load data
-    data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False)
+    if pack_version.Version(pysat.__version__) < pack_version.Version('3.0.2'):
+        data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False,
+                                              epoch_name='ut')
+
+        # Create datetimes from 'ut' variable
+        data['time'] = [dt.datetime(2019, 1, 1)
+                        + dt.timedelta(seconds=int(val * 3600.0))
+                        for val in data['ut'].values]
+    else:
+        epoch = dt.datetime(2019, 1, 1)
```

Comment:
Shouldn't the date be coming from the Instrument module?

Reply:
I think the epoch origin for the Sami files is constant, or at least that was how the code was when I modified for file loading compatibility. At first I interpreted the statement differently and moved the epoch_origin to a Sami module attribute. Anyway, we should check with @jklenzing to make sure a constant epoch of 2019, 1, 1 is correct.